test/astar.cc

```diff
@@ -1570,9 +1570,8 @@ TEST(BiDiAstar, test_recost_path) {
            3  4        5
   )";
   const gurka::ways ways = {
-      // make ABC to be a shortcut
+      // make ABCDE to be a shortcut
```

Comment:
somehow this test is still failing.. it now takes the supposedly longer route, not ABCDE and I don't understand why yet..